packages/quantic/force-app/main/default/lwc/quanticHeadlessLoader/quanticHeadlessLoader.js

```diff
@@ -8,18 +8,32 @@ const DEBOUNCE_DELAY = 200;
 let debouncers = {};
 let dependencyPromises = [];
 
+
+const headlessBundles = {
```

Comment:
What do you think of exporting an enum or just a plain object for the use case names so implementers won't have to write out the string.

Reply:
Good point, I'll look into it.